ROADMAP.md

```diff
@@ -1,40 +1,43 @@
 # Roadmap
 
-Currently, builders can:
+This document summarizes current state for the Sui blockchain and hints at impending changes. For the latest updates, see:
+https://docs.sui.io/devnet/learn#see-whats-new
+
+Currently, Sui builders can:
+* Connect to Sui Devnet
 * Write and test Move smart contracts
-* Spin up a local Sui network
-* Publish and run Move smart contracts on a local network
+* Publish and run Move smart contracts
+* Program with Sui objects
+* Run a Sui Fullnode
+* Use Sui Explorer to see transactions
+
+See instructions for all of the above at:
+https://docs.sui.io
 
 In the coming months, we will release:
-1. A public devnet that allows Sui devs to do all of the above on a shared network powered by Mysten-operated authorities
```

Comment:
this is not rendering as a list in preview, and starts with 2. 

<img width="1119" alt="image" src="https://user-images.githubusercontent.com/109545725/185184837-9f92397b-64b3-4c10-bb45-cae775dde7f3.png">

Reply:
fixed with a new commit